### PR TITLE
SSR documentation clarification

### PR DESCRIPTION
**Pull request by scarcanague**
Clarifying  the code biases signal encoding and correcting a few nits

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -626,7 +626,7 @@
     },
     "node_modules/assert/node_modules/util": {
       "version": "0.10.3",
-      "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+      "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```
Comment (richarddeurloo) [inline package-lock.json:629]: Is this correct? I'd expect all addresses to be http**_S_**

File: package-lock.json [hunk 2]
```diff
@@ -7523,7 +7522,7 @@
         },
         "util": {
           "version": "0.10.3",
-          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+          "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```
Comment (richarddeurloo) [inline package-lock.json:7525]: Same as above.

**Review by richarddeurloo — COMMENTED** [review]
Just wondering about the updated hyperlink.

**Review by ljbade — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (scarcanague) [reply package-lock.json:629]: Hum that's a good point. This is automatically generated but Ill ping the `libsbp` channel to see if we can update the generator

**Pull request merged**